src/components/Chat/InputMessageV2.tsx

```diff
@@ -194,24 +401,44 @@ const InputMessageV2 = ({
             labelLeft={!inputFocus ? 'Anonymity' : undefined}
             styleLabelLeft={[
               styles.labelToggle,
-              {color: type === 'ANONYMOUS' ? colors.gray : colors.blue}
+              {color: type === ANONYMOUS ? colors.gray : colors.blue}
             ]}
-            onValueChange={toggleChnage}
-            value={type === 'ANONYMOUS'}
+            onValueChange={toggleChange}
+            value={type === ANONYMOUS}
           />
         )}
       </View>
-      <TouchableOpacity
-        style={[styles.btn, isDisableButton() ? styles.disableButton : styles.enableButton]}
-        disabled={isDisableButton()}
-        onPress={handleSendMessage}>
-        <IconSend
-          style={styles.icSendButton}
-          fillBackground={sendButtonStyle()}
-          fillIcon={colors.white}
-        />
-      </TouchableOpacity>
-      <SheetEmoji ref={refEmoji} selectEmoji={onSelectEmoji} />
+      {text?.trim() !== '' && (
+        <TouchableOpacity
+          style={[styles.btn, isDisableButton() ? styles.disableButton : styles.enableButton]}
+          disabled={isDisableButton()}
+          onPress={handleSendMessage}>
+          <IconSend
+            style={styles.icSendButton}
+            fillBackground={sendButtonStyle()}
+            fillIcon={colors.white}
+          />
+        </TouchableOpacity>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The send button now sends an empty attachments array if there's no text in the input field. Make sure that the backend can handle an empty attachments array and that this doesn't cause any unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->